src/Neo/Wallets/Wallet.cs

```diff
@@ -136,6 +136,26 @@ protected Wallet(string path, ProtocolSettings settings)
             Path = path;
         }
 
+        public WalletAccount CreateMultiSigAccount(params ECPoint[] publicKeys) =>
+            CreateMultiSigAccount(publicKeys.Length, publicKeys);
```

Comment:
I'd expect BFT number of keys here, but maybe that's just me.

See https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/smartcontract#CreateDefaultMultiSigRedeemScript also.

Reply:
That's exactly what I'm talking about, expectations of `CreateMultiSigAccount(keys)` caller.

Reply:
My suggestion is `s,publicKeys.Length,publicKeys.Length - (publicKeys.Length-1)/3,`.